doc/scheduler-scripting.tex

```diff
@@ -621,20 +622,23 @@ \subsubsection{Jupyter Notebooks}
 
 \end{enumerate}
 
-\begin{figure}[htbp]
+\begin{figure}[ht]
 	\centering
-	\fbox{\includegraphics{images/putty1}}
```

Comment:
Please keep `\fbox`, it gives good frame around the white spots of the image.

Reply:
fixed